README.md

```diff
@@ -155,6 +157,75 @@ if __name__ == '__main__':
     IOLoop.current().run_sync(main)
 ```
 
+## `notebook-http` Mode
+
+The `KernelGatewayApp.api` command line argument can be set to `notebook-http`
+to allow a properly annotated notebook to be served via HTTP endpoints. A
+notebook cell must be annotated with a single line comment at the beginning
+describing the HTTP endpoint. An example python cell is:
+
+```python
+# GET /hello/world
+print("hello world")
+```
+
```

Comment:
Would be good to mention somewhere that this feature *should* support comment syntax in any language but that's its a work in progress to identify them all.